osbenchmark/client.py

```diff
@@ -210,6 +212,24 @@ def __init__(self, hosts, client_options):
         if self._is_set(self.client_options, "enable_cleanup_closed"):
             self.client_options["enable_cleanup_closed"] = convert.to_bool(self.client_options.pop("enable_cleanup_closed"))
 
+    @staticmethod
+    def _has_only_hostnames(hosts):
```

Comment:
is the the host names from `--target-host` parameter? @IanHoang

Reply:
@rishabh6788 Yes, that is correct.